sources/packages/api/src/services/disbursement-receipt/disbursement-receipt.service.ts

```diff
@@ -123,4 +123,38 @@ export class DisbursementReceiptService extends RecordDataModelService<Disbursem
       .getRawOne();
     return batchRunDate?.max ?? new Date();
   }
+
+  /**
+   * Get the disbursement receipt details for
+   * given assessment.
+   * @param assessmentId assessment to which disbursement
+   * receipt belongs to.
+   * @param studentId student to whom the disbursement
+   * receipt belongs to.
+   * @returns disbursement receipt details.
+   */
+  async getDisbursementReceiptByAssessment(
+    assessmentId: number,
+    studentId?: number,
+  ): Promise<DisbursementReceipt[]> {
+    return this.repo.find({
+      select: {
+        id: true,
+        disbursementSchedule: { id: true },
+        disbursementReceiptValues: { grantType: true, grantAmount: true },
+      },
+      relations: {
```

Comment:
do we need the relations for disbursementSchedule, if we are selecting the id:true in the select? Can we have a call to check?

Reply:
As discussed in the call, it did not work without using relations.